packages/indexer/src/sinks/drizzle/drizzle.ts

```diff
@@ -45,11 +69,17 @@ export class DrizzleSink<
 
   async transaction(
     cb: (params: {
-      db: PgTransaction<TQueryResult, TFullSchema, TSchema>;
+      transaction: (
+        endCursor?: Cursor,
```

Comment:
Notice that `endCursor` is already available here and comes from `message.data.endCursor`.